h/presenters.py

```diff
@@ -68,6 +68,29 @@ def uri(self):
         return jinja2.escape(self.annotation.target_uri)
 
     @property
+    def text(self):
+        """The body text of this annotation.
+
+        """
+
+        return  jinja2.escape(self.annotation.text)
+
+    @property
+    def quote(self):
+        """The target text of this annotation.
+
+        """
+        def get_selection():
```

Comment:
This could be extracted into a `_quote()` helper method on the class and the `description()` method below refactored accordingly.